docs/serving/samples/hello-world/helloworld-php/README.md

```diff
@@ -57,6 +57,14 @@ cd knative-docs/serving/samples/hello-world/helloworld-php
    RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
    ```
 
+1. Create a `.dockerignore` file to ensure that any files related to a local build, do not affect the container that you build for deployment.
+
+   ```ignore
```

Comment:
@RichieEscarez 

Is there any way to do some fancy server-side include magic here rather than needing to repeat the content?

Reply:
for the rendered content on the site, yes we have a few options (ie. using Hugo variables to inject chunks of content). the downside is that when the content is viewed in GitHub, its no longer illegible (more difficult for authors or one-off contributors, especially those who use the GitHub WYSIWYG UI). im toying with the idea of getting our Netlify builds to run on demand (therefore the authoring process / PR review and merge steps would include running a "test build" to render and verify the output) but that too adds to the overhead of contributing. I think the first place we want to start to implement "content reuse" and "single sourcing" are for the multiple install guides

we should chat more if you have ideas here?

Reply:
> for the rendered content on the site, yes we have a few options (ie. using Hugo variables to inject chunks of content). the downside is that when the content is viewed in GitHub, its no longer legible (more difficult for authors or one-off contributors, especially those who use the GitHub WYSIWYG UI). im toying with the idea of getting our Netlify builds to run on demand (therefore the authoring process / PR review and merge steps would include running a "test build" to render and verify the output) but that too adds to the overhead of contributing. I think the first place we want to start to implement "content reuse" and "single sourcing" are for the multiple install guides
> 
> we should chat more if you have ideas here?

I would like something like [SSI Include syntax](https://www.w3.org/Jigsaw/Doc/User/SSI.html#include):
```
<!--#include file="included.html" -->
```
Which seems like it would be fairly natural to read. But I don't know if Hugo has that capability.